sjb/config/common/test_cases/origin_release_install_azure_39.yml

```diff
@@ -163,6 +164,36 @@ extensions:
           -e "openshift_azure_input_image_prefix=centos7" \
           playbooks/azure/openshift-cluster/launch.yml
 
+    - type: "script"
+      title: "run the acs-engine e2e test"
+      script: |-
+        set -a +o xtrace
+        . /data/src/github.com/openshift/release/cluster/test-deploy/azure/credentials
+        set +a -o xtrace
+
+        cd /data/src/github.com/Azure/acs-engine
```

Comment:
do you just need to docker run -v /data/src/github.com/Azure/acs-engine:/go/src/github.com/Azure/acs-engine:z ?
can you avoid the make build outside the docker container?

Reply:
Does not look like a blocker